forge/forge/op/eval/forge/convolution.py

```diff
@@ -226,57 +226,52 @@ def eval(self, tensors):
         assert len(tensors) <= 3, "ConvTranspose ops should have up to three inputs (input, weight, bias)"
         assert len(tensors) >= 2, "ConvTranspose ops should have at least two inputs (input, weight)"
         t_ops = to_torch_operands(*tensors)
-
         activations = t_ops[0]
         weights = t_ops[1]
         bias = t_ops[2] if len(t_ops) == 3 else None
 
         stride = [self.stride_height, self.stride_width]
         dilation = [self.dilation_height, self.dilation_width]
         groups = self.groups
-        padding = [
-            self.padding_left,
-            self.padding_right,
-            self.padding_top,
-            self.padding_bottom,
-        ]
+        # TODO: Add support for asymmetric padding cases in convtranspose2d
+        # https://github.com/tenstorrent/tt-forge-fe/issues/665
+        if self.padding_left == self.padding_right and self.padding_top == self.padding_bottom:
+            padding = (self.padding_top, self.padding_left)
+        else:
+            assert self.padding_left == self.padding_right, "Padding left and Padding right must be equal"
```

Comment:
Just update the message like: "Currently, different left and right padding isn't supported. Please checkout this issue (issue link) for more details".